### ISSUE TITLE
[Feature] Add option to change video resolution/sound quality

### ISSUE THREAD

**Issue by Quix0r** [dino/dino#1093]
Not all people around the world have enough bandwidth, e.g. in Zambia I know it from my girlfriend and I saw it first-hand that the Internet is often not enough for 1080p+ videos, you really have to go down a lot further, even sometimes below VGA (640x480).

Can you please add an option/dialog box/tab for video/audio quality settings, e.g. like Linphone has?

**Comment by licaon-kter** [comment]
Aren't codec/resolution negotiated by the underlying webrtc lib to the best available?

Orientation is determined by device, right?

**Comment by Quix0r** [comment]
Yes, maybe that part does auto-detect the "best" resolution (that you can squeeze in the available bandwidth) but it is not always the best choice when it comes to Internet bundles. Look, we both stream with 1080p and 44.1 kHz, that's going to use quite a lot bandwidth, if you ask me (my guess).

It would be nice if there is a manual way to lower/raise the resolution/audio quality or even change codecs to low-bandwidth codecs (like VP8?).
